### ISSUE TITLE
[CoE Starter Kit - QUESTION] QUESTION

### ISSUE THREAD

**Issue by sanjeev40084** [microsoft/coe-starter-kit#4926]
### What is your question?

We are updating our CoE solutions which hasn't been updated for a year. When we first setup CoE we deployed the power bi dashboard that was available with the release i.e. Product_CoEDashboard_May2022.pbit. With the current (Feb) release, it looks like there is updated powerbi report i.e. Production_CoEDashboard_February2023.pbit. So my question is if we publish the new report we will end up with 2 reports one with May and another with February title. Is that how it is supposed to be or are we supposed to rename the report somewhere so that whenever there is update on report with every new release, we just rename and after publish it just updates the same report ? Sorry if this seems such a generic task, still newbie on powerbi world. :)

### What solution are you experiencing the issue with?

None

### What solution version are you using?

February 2023

**Comment by Jenefer-Monroe** [comment]
Hello. You may not succeed in attempting to upgrade a CoE envt from a year ago. We recommend upgrading at least every 3 months as that is as far back as we test. Please see [Updating the Center of Excellence (CoE) Starter Kit](https://learn.microsoft.com/power-platform/guidance/coe/after-setup)

Regarding the PBI's. The idea is that when you save, you would save and publish as some consistent name (My Org PP CoE.pbix). When you save and publish the next version to the same workspace this way it will ask you if you want to overwrite the old one. Thats how people have been doing this. 

I hope that helps!

**Comment by sanjeev40084** [comment]
@Jenefer-Monroe thanks for the info. We will follow your instruction regarding saving and publishing the power bi report. P.S: we were able to upgrade to new CoE instance. Only the issue we are running into is, we are seeing some stale data (like it reports on flows and apps which don't even exist on environment anymore) so not sure how to get that corrected. Since the sync was turned off for most of the time, are we supposed to run full inventory sync now?

**Comment by Jenefer-Monroe** [comment]
Yes you will need to do a full inventory run. Please see [Running a full inventory](https://learn.microsoft.com/power-platform/guidance/coe/faq#running-a-full-inventory)

Once that is done, please note that there are helper flows to do things like clean up deleted items. 
You can use the CoE Admin Command Center to make sure all the inventory flows are running, and when they last ran / passed. You can sort by State to see this easily. 

![image](https://user-images.githubusercontent.com/89584709/223733701-6d94235c-36ff-42ff-aebb-ce88f096e901.png)

And filter to the cleanups to make sure they have been running (do this after your full inventory run)
![image](https://user-images.githubusercontent.com/89584709/223734027-5c5a2642-5cf9-444a-95d8-5ea5532fca86.png)

**Comment by sanjeev40084** [comment]
@Jenefer-Monroe thanks for the info. 👍
